styles.css

```diff
@@ -0,0 +1,73 @@
+
```

Comment:
you've got two style.css files. It's not clear to me what they're for. Maybe it's obvious when looking at all your code at once (github hides the full path) but from what I'm looking at, I'd suggest giving them different names to hint at what they're for.